src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/ImplementIDisposableCorrectlyTests.cs

```diff
@@ -1781,8 +1768,7 @@ Inherits B
         MyBase.Finalize()
     End Sub
 End Class|]
-",
-            GetCA1063BasicFinalizeOverrideResultAt(11, 14, "C"));
+");
```

Comment:
Unrelated to your change, but I wonder if this should fire FinalizeImplementation rule as it doesn't invoke Dispose(False)